homeassistant/components/laundrify/translations/en.json

```diff
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "abort": {
+      "single_instance_allowed": "Already configured. Only a single configuration possible."
+    },
+    "error": {
+      "cannot_connect": "Failed to connect",
+      "invalid_auth": "Invalid authentication",
+      "invalid_format": "Invalid format. Please specify as xxx-xxx.",
+      "unknown": "Unexpected error"
+    },
+    "step": {
+      "init": {
+        "data": {
+          "code": "Auth Code (xxx-xxx)",
+          "poll_interval": "Poll Interval (in seconds)"
```

Comment:
remove

Reply:
Removed in 95018cedae969e535cba4f82c75d63664d162291